discovery-infra/test_infra/helper_classes/nodes.py

```diff
@@ -41,6 +44,24 @@ def __iter__(self) -> Iterator[Node]:
         for n in self.nodes:
             yield n
 
+    @classmethod
+    def _get_namespace_index(cls, libvirt_network_if):
+        # Hack to retrieve namespace index - does not exist in tests
+        matcher = re.match(r'^tt(\d+)$', libvirt_network_if)
+        return int(matcher.groups()[0]) if matcher is not None else 0
+
+    def configure_nat(self):
+        libvirt_network_if = self.controller.network_conf.libvirt_network_if
```

Comment:
Maybe create

```
@property
def nat_interfaces(self):
	return (self.controller.network_conf.libvirt_network_if, self.controller.network_conf.libvirt_secondary_network_if)
```

Then the configure functions become simpler:
```
def configure_nat(self):
        self._nat_controller.add_nat_rules(self.nat_interfaces, self._get_namespace_index(libvirt_network_if))

    def unconfigure_nat(self):
        self._nat_controller.remove_nat_rules(self.nat_interfaces, self._get_namespace_index(libvirt_network_if))
```

Reply:
Oh sorry, didn't notice you just copy pasted that code. It would still be nice to change that if you're already changing this code